scanner/src/main/kotlin/scanners/fossid/FossId.kt

```diff
@@ -186,6 +196,23 @@ class FossId(
 
         log.info { "waitForResult parameter is set to '$waitForResult'" }
 
+        val namingProjectPattern = fossIdScannerOptions["namingProjectPattern"]
+        val namingScanPattern = fossIdScannerOptions["namingScanPattern"]
+
+        val namingConventionVariables = fossIdScannerOptions
+            .filterKeys { it.startsWith(NAMING_CONVENTION_VARIABLE_PREFIX) }
+            .mapKeys { it.key.substringAfter(NAMING_CONVENTION_VARIABLE_PREFIX) }
```

Comment:
From just reading the code, this looks strange: You're basically just stripping prefixed from key names. Which raises the question: Why bother adding these prefixes in the first place if they get stripped anyway?

Reply:
Please see my answer above to Oliver:

> I did so to be able to group everything naming convention-related in the ort.conf, since it's no possible to have structured data